src/graph.ts

```diff
@@ -236,7 +242,11 @@ export class Graph {
       outgoing_buttons: [],
       incoming_buttons: [],
     }
-    inner.html = GraphHTML.renderModule(id, inner);
+    if(typeof top == undefined){
```

Comment:
`typeof top == 'undefined'` -- JavaScript has horrible edge cases for checking null/undefined equality...